config/README.md

```diff
@@ -1,102 +1,103 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
+
+## Preface
+
+This doc provides a well-lit path for deploying the DSS and its dependencies (
+namely CockroachDB) on Kubernetes. This is not a requirement, and a DSS node
+can join a cluster as long as it meets the following requirements:
+
+*   Every CockroachDB node must advertise a unique and routable address.
+    *   It is preferred to use domain names with unique prefixes and homogenous
+        suffixes, ie: 0.c.dss.interussplatform.com.
+    *   This allows wildcard usage in the CRDB certificates.
+*   Every DSS node should run a minimum of 3 CockroachDB nodes.
+*   At least 3 CockroacbDB addresses must be shared with all participants.
+    *   If not using the recommended hostname prefix above, every cockroachDB
+        hostname must be shared with every participant.
+*   Every DSS node must supply and share their CockroachDB public certificate
+*   All CockroachDB nodes must be run in fully secure mode.
+*   The ordering of the `--locality` flag keys must be the same across all
+    CockroachDB nodes in the cluster.
+
+Note: we are investigating the use of service mesh frameworks to alleviate some
+of this operational overhead.
 
 ## Prerequisites
 
-Download & install:
+Download & install the following tools to your workstation:
 
-*   helm
+*   The [helm client](https://helm.sh/docs/using_helm/#installing-the-helm-client)
+    *   Required if deploying using the defined Kubernetes templates.
 *   kubectl
+    *   Required if deploying with Kubernetes.
 *   docker
-*   cockroachdb
+    *   Required if building new images of the DSS.
+*   Cockroachdb
+    *   Required to generate new CockroachDB certificates.
 *   Google Cloud SDK (if deploying on GCP)
-*   [Optional] Golang. Recommended to understand go, and the go toolchain.
+    * Required if deploying to Google Cloud.
+*   Golang.
+    *   Required if developing the DSS codebase.
 
 
 ## Building Docker images
```

Comment:
This heading should probably now be just "Docker images" with a "Building Docker images" subheading directly preceding the "Set environment variable..." line since the first paragraph here does not relate to building images.

Reply:
done